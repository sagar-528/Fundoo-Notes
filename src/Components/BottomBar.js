import React, { Component } from 'react'
import  { View } from 'react-native'
import { Appbar, FAB } from 'react-native-paper';
import HomeCss from '../Styles/HomeCss';

export class BottomBar extends Component {
    render() {
        return (
            <View>
                <Appbar
                style = {HomeCss.bottom}>
                    <Appbar.Action icon = "check-box-outline" />
                    <Appbar.Action icon = "draw"  />
                    <Appbar.Action icon = "microphone-outline"  />
                    <Appbar.Action icon = "panorama"  />
                </Appbar>
                <FAB icon = "plus"  style = {HomeCss.fab} />
            </View>
        )
    }
}


export default BottomBar
