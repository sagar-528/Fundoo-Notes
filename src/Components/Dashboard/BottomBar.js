import React, { Component } from 'react'
import  { View } from 'react-native'
import { Appbar, FAB } from 'react-native-paper';
import NoteCss from '../../Styles/NoteCss'

export class BottomBar extends Component {
    render() {
        return (
            <View>
                <Appbar
                style = {NoteCss.bottom}>
                    <Appbar.Action icon = "check-box-outline" />
                    <Appbar.Action icon = "brush"  />
                    <Appbar.Action icon = "microphone-outline"  />
                    <Appbar.Action icon = "panorama"  />
                </Appbar>
                <FAB icon = "plus"  
                style = {NoteCss.fab}
                onPress = {() => this.props.navigation.navigate('AddNote')}
                />
            </View>
        )
    }
}


export default BottomBar
