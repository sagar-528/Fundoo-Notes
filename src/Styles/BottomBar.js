import { StyleSheet } from 'react-native';

const BottomBarStyle = StyleSheet.create({

  bottombar_view : {
    width: '100%',  
    backgroundColor : 'white',
    position: 'absolute',
    bottom : 0,
  },

  bottombar : {
    backgroundColor : 'white',
    borderWidth : 1,
    borderColor : '#CACFD2',
},

plus_button_style : {
    marginRight : '10%', 
    borderWidth : 2, 
    marginBottom : 60, 
    height : 60, 
    width : 60, 
    backgroundColor : '#e0e0e0', 
    borderRadius : 30, 
    borderColor : '#e0e0e0',
},

bottom_bar_action_style : {
    marginRight : 10,
}

})

export default BottomBarStyle