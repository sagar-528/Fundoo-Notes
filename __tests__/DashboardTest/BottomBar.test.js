import React from 'react';
import {shallow} from 'enzyme';
import BottomBar from '../../src/Components/Dashboard/BottomBar'

describe('test BottomBar', () => {
    
    it('should match to snapshot', () => {
        const component = shallow(<BottomBar />)
        expect(component).toMatchSnapshot();
    })

    it('test appbar action and icon in bottom bar component', () => {
        const component = shallow(<BottomBar />)
        expect(component.find(Appbar.Action)).toHaveLength(5)
        expect(component.find(Appbar.Action).at(0).props().icon).toEqual('check-box-outline')
        expect(component.find(Appbar.Action).at(1).props().icon).toEqual('brush')
        expect(component.find(Appbar.Action).at(2).props().icon).toEqual('microphone-outline')
        expect(component.find(Appbar.Action).at(3).props().icon).toEqual('panorama')
        expect(component.find(Appbar.Action).at(4).props().icon).toEqual('plus')
    })
    
})