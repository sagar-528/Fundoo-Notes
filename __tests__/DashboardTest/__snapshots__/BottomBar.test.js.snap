// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test BottomBar should match to snapshot 1`] = `ShallowWrapper {}`;
